local-test.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs')
+const Firestore = require('./utils/firestore.js')
+const config = require('config')
+
+// check whether github credentials are not falsy
+if (
```

Comment:
Will it be a good thing if we wrap this thing in a function and then call the function within readWriteCheck? We can do this for both GitHub/ firebase validations.

Reply:
Reason - we can unit test the validations. Also, we will be knowing where validations are required explicitly.

Reply:
I want to know more. Can we come talk for few minutes on discord?

Reply:
yes sure